index.js

```diff
@@ -1,14 +1,16 @@
 'use strict';
-module.exports = function (a, b) {
+module.exports = function (a, b, minComp) {
 	if (!Buffer.isBuffer(a) || !Buffer.isBuffer(b)) {
 		throw new TypeError('Arguments must be Buffers');
 	}
 
+	var len = Math.max(a.length, b.length, minComp || 0);
```

Comment:
If we throw an exception, then the implementer have to handle it or may use `buffer-equals`. Therefore the implementer is aware about a potential information leak.

Having reflected on it, I would prefer throwing. `bufferEqualsConstant` can't be _constant_ in this situation, without returning a false value.